lib/utils/qemu.coffee

```diff
@@ -34,7 +34,9 @@ exports.copyQemu = (context, arch) ->
 		new Promise (resolve, reject) ->
 			read = fs.createReadStream(qemu)
 			write = fs.createWriteStream(binPath)
+
 			read
+			.on('error', reject)
```

Comment:
Do we really need this one as well?
Wouldn't `reject` end-up being called twice?